code/obj/item/playing_cards.dm

```diff
@@ -1,1137 +1,1265 @@
+            def = rand(0, 10)
+            if(chosen_card_type.LVL)
+                atk *= chosen_card_type.LVL
+                def *= chosen_card_type.LVL
+        else
+            atk = chosen_card_type.ATK
+            def = chosen_card_type.DEF
+
+        name += " [atk]/[def]"
+        desc = chosen_card_type.card_data
+        desc += " ATK [atk] | DEF [def]"
+
+    proc/stg_friend(var/list/possible_card_types)
+        var/path = pick(possible_card_types)
+        var/datum/playing_card/griffening/creature/friend/chosen_card_type = new path
+        if(chosen_card_type.LVL)
+            name = "LVL [chosen_card_type.LVL] [chosen_card_type.card_name]"
+        else
+            name = chosen_card_type.card_name
+        var/atk
+        var/def
+        if(chosen_card_type.randomized_stats)
+            atk = rand(0, 10)
+            def = rand(0, 10)
+            if(chosen_card_type.LVL)
+                atk *= chosen_card_type.LVL
+                def *= chosen_card_type.LVL
+        else
+            atk = chosen_card_type.ATK
+            def = chosen_card_type.DEF
+        name += " [atk]/[def]"
+        desc = chosen_card_type.card_data
+        desc += " ATK [atk] | DEF [def]"
+        icon_state = "stg-general-[rand(1,NUMBER_GENERAL)]"
+
+    proc/stg_effect(var/list/possible_card_types)
+        var/path = pick(possible_card_types)
+        var/datum/playing_card/griffening/effect/chosen_card_type = new path
+
+        name = chosen_card_type.card_name
+        desc = chosen_card_type.card_data
+        icon_state = "stg-general-[rand(1,NUMBER_GENERAL)]"
+
+    proc/stg_area(var/list/possible_card_types)
+        var/path = pick(possible_card_types)
+        var/datum/playing_card/griffening/area/chosen_card_type = new path
+
+        name = chosen_card_type.card_name
+        desc = chosen_card_type.card_data
+        icon_state = "stg-general-[rand(1,NUMBER_GENERAL)]"
+
+    proc/add_foil() //makes the card shiiiiiny
+        UpdateOverlays(image(icon,"stg-foil"),"foil")
+        foiled = TRUE
+        name = "foil [name]"
+
+/obj/item/playing_card/expensive //(¬‿¬)
+    desc = "Tap this card and sacrifice one Yourself to win target game."
+    icon_state = "stg-general-0"
+    var/list/prefix1 = list("Incredibly", "Strange", "Mysterious", "Suspicious", "Scary")
+    var/list/prefix2 = list("Rare", "Black", "Dark", "Shadowy", "Expensive", "Fun", "Gamer")
+    var/list/names = list("Flower", "Blossom", "Tulip", "Daisy")
+
+    New()
+        ..()
+        name = "[pick(prefix1)] [pick(prefix2)] [pick(names)]"
+        update_stored_info()
+
+    MouseDrop(var/atom/target as obj|mob)
+        ..()
+        if(tapped)
+            var/mob/user = usr
+            user.deathConfetti()
+            playsound(get_turf(user.loc), 'sound/musical_instruments/Bikehorn_1.ogg', 50)
+            user.visible_message("<span class='combat'><b>[uppertext(user.name)] WINS THE GAME!</b></span>")
+            if(!foiled)
```

Comment:
how would this become foiled? it's not set in the definition

Reply:
it rolls a random chance in it's new